crates/typst/src/foundations/content.rs

```diff
@@ -374,6 +374,16 @@ impl Content {
         Self::sequence(std::iter::repeat_with(|| self.clone()).take(count))
     }
 
+    /// Retrive styled content's child (the content alone).
+    pub fn styled_child(&mut self) -> &mut Content {
+        // Awaiting Polonius...
+        if self.is::<StyledElem>() {
+            let styled_elem = self.to_packed_mut::<StyledElem>().unwrap();
```

Comment:
I assume destructuring a call to `to()` lead to some problem with lifetimes?

Reply:
Yep.